store/store_test.go

```diff
@@ -2003,7 +2003,7 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreRepositoryDetails(c *C) {
 	c.Check(result.Architectures, DeepEquals, []string{"all"})
 	c.Check(result.Revision, Equals, snap.R(27))
 	c.Check(result.SnapID, Equals, helloWorldSnapID)
-	c.Check(result.Publisher, Equals, "canonical")
+	c.Check(result.Publisher, Equals, "Canonical")
```

Comment:
This broke tests:
```
in test-snapd-python-webserver.publisher expected 'canonical', got 'Canonical'
```

Reply:
Actually it's https://github.com/snapcore/snapd/pull/4532/files#diff-335c37cf601209543ed44f0aeca1073aR94 that broke